script.js

```diff
@@ -46,6 +46,260 @@ function createComponent(templateId, template) {
+    third_dimension_model_text: null,
+    third_dimension_model_image: '/images/moses/desktop_card.png'
+  },
+  'pasterski': {
+    header_image: '/images/pasterski/desktop_banner.png',
+    title: 'Dr. Sabrina Gonzalez Pasterski',
+    description: "Sabrina Pasterski is a proud first-generation Cuban-American & Chicago Public Schools alumna who, by 16, had built & flown her own airplane. She became the first MIT freshman named to the NASA January Operational Internship and earned the inaugural MIT Freshman Entrepreneurship Award. After a summer at Phantom-Works, excelling at MIT and being hired by CERN-CMS, she was named a Lindau Nobel Young Researcher. She graduated #1 at MIT-Physics, matriculated into Harvard’s PhD program and was awarded her SB while still a teenager. Sabrina was named to Scientific American’s 30 under 30 in 2012 as well as the 2015 Forbes 30 under 30 Science list. <br /><br /> At age twelve, Sabrina started building a two-seat one-hundred horsepower light sport aircraft. At fourteen, she flew her Cessna 150 solo in Canada and completed the build of her experimental aircraft. She conducted her first US solo flight in the airplane she built at sixteen and graduated MIT while still a teenager. She earned her PhD at Harvard under Andy Strominger. Half way through her post-doc at the Princeton Center for Theoretical Science, she was targeted by Brown University and the Perimeter Institute for open faculty positions—she declined Brown and accepted Perimeter becoming a professor of theoretical physics at age twenty-seven.",
+    third_dimension_model_text: null,
+    third_dimension_model_image: '/images/pasterski/desktop_card.png'
+  },
+  'wood': {
+    header_image: '/images/wood/desktop_banner.png',
+    title: 'Dr. Danielle Wood',
+    description: null,
+    third_dimension_model_text: "This model shows a pyramid with numerous images regarding NASA and space, which are heavy influences in Danielle Wood’s career as an Assistant Professor in Media Arts & Sciences at MIT. <br /> Within the MIT Media Lab, Prof. Wood leads the Space Enabled Research Group which seeks to advance justice in Earth's complex systems, using designs enabled by space. <br /> Wood is a scholar of societal development with a background that includes satellite design, earth science applications, systems engineering, and technology policy. In her research, Prof. Wood applies these skills to design innovative systems that harness space technology to address development challenges around the world.",
+    third_dimension_model_image: '/images/wood/desktop_card.png'
+  }
+};
+genius_card = document.querySelector("genius-card");
+if (genius_card.getAtrribute("onclick") == true) {
+  switch (content) {
+      case "joy":
+        location.href = location.href + "/joy";
+        insertImage(header_image, content[0][header_image], header);
```

Comment:
Since `content` is an object, I don't think using array based indexing would work here. Does it work for you?